front/src/components/Detail.jsx

```diff
@@ -30,7 +29,8 @@ const Detail = () => {
       <h1 style={{ color: "blue", fontSize: "18px" }}>{pageQuestion.text}</h1>
       {pageChoices ? (
         pageChoices.map((item) => (
-          <h1 style={{ color: "black", fontSize: "16px" }}>
+          <h1 style={{ color: "black", fontSize: "16px" }}
```

Comment:
Has this been linted and formatted? I don't think this should pass pretter/eslint. Did we set you up with front end linting yet?

Reply:
No, I have not linted since this last PR. I should be linting before every commit and will do so going forward. Checking now, no these lines of code def do not pass eslint.